Library/Homebrew/extend/os/linux/extend/ENV/super.rb

```diff
@@ -1,13 +1,26 @@
 module Superenv
+  alias x11? x11
+
   # @private
   def self.bin
     (HOMEBREW_SHIMS_PATH/"linux/super").realpath
   end
 
+  def xorg_recursive_deps
+    if xorg_installed?
+      @xorg_deps ||= Formula["linuxbrew/xorg/xorg"].recursive_dependencies.map(&:to_formula)
```

Comment:
This is getting outside of my Ruby comfort zone, but you could try a class variable.
```ruby
@@xorg_deps ||= …
```